feature/isis/otg_tests/weighted_ecmp_test/metadata.textproto

```diff
@@ -35,3 +35,13 @@ platform_exceptions: {
     weighted_ecmp_fixed_packet_verification: true
   }
 }
+platform_exceptions: {
```

Comment:
Why the changes added for CISCO vendor

Reply:
This is a result of merging main into branch -- see 1498992